### ISSUE TITLE
Add explanation of work segment codes in user guide

### ISSUE THREAD

**Issue by binnympaul** [RSGInc/SOABM#30]
work segment codes in the wsLoc.csv output file mean the following:
-1: Not a worker
0 : Management
1 : White Collar
2 : Blue Collar
3 : Sales and..
4 : Natural..
5: Production,..
99999 : Works from home

**Comment by bstabler** [comment]
add wsLoc table to outputs and describe each field:


HHID | HomeMGRA | Income | PersonID | PersonNum | PersonType | PersonAge | EmploymentCategory | StudentCategory | WorkSegment | SchoolSegment | WorkLocation | WorkLocationDistance | WorkLocationLogsum | SchoolLocation | SchoolLocationDistance | SchoolLocationLogsum
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
48228 | 1 | 111249 | 114320 | 1 | 1 | 40 | 1 | 3 | 3 | -1 | 7 | 2.99E+00 | -1.87E-01 | 0 | 0.00E+00 | -9.99E+02
48228 | 1 | 111249 | 114321 | 2 | 1 | 42 | 1 | 3 | 3 | -1 | 1423 | 7.18E+00 | -6.22E-01 | 0 | 0.00E+00 | -9.99E+02
48229 | 1 | 71362 | 114322 | 1 | 4 | 50 | 3 | 3 | -1 | -1 | 0 | 0.00E+00 | -9.99E+02 | 0 | 0.00E+00 | -9.99E+02

**Comment by binnympaul** [comment]
Here is what I found for the following variables:
Employment category (1-employed FT, 2-employed PT, 3-not employed, 4-under age 16)
Student category (1 - student in grade or high school; 2 - student in college or higher; 3 - not a student)
School segment is "-1" for all non-students, 0 for all pre-schoolers, 1-8: grade school school district, 9-16: HS school districts, 17-18: typical an non-typical university students

School segment and school location is "88888" for few students. I looked into Java code and could not understand why they are set to "88888". I believe they might be home-schooled.
